dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CommonWebServerTests15Servlet40.java

```diff
@@ -26,18 +26,26 @@
 @RunWith(FATRunner.class)
 @Mode(FULL)
 public class CommonWebServerTests15Servlet40 extends CommonWebServerTests {
-    @AfterClass
-    public static void stopTestServer() throws Exception {
+
+    @After
```

Comment:
I assume we now intend to restart the server between each test.  I thought we had some tests that did not want to do that.

Reply:
Those tests are in CommonWebServerTests15.java